### ISSUE TITLE
GitHub console note opening

### ISSUE THREAD

**Issue by SteveALee** [madskristensen/OpenCommandLine#31]
I switched to the GitHub console but Alt+space then does nothing (works fine with cmd).

I'm using VS community 2015 and latest GitHub installed "Never Gonna Git You Up (3.0.7.1) 3397ae4" 
Sorry to Rickroll you - haha

Is it something to do with there change of name to GitHub Desktop?

**Comment by madskristensen** [comment]
Yes, they may have changed how the GitHub Desktop app lays down the console executable.
